app/monitoringapi.go

```diff
@@ -106,7 +106,12 @@ func startReadyChecker(ctx context.Context, tcpNode host.Host, eth2Cl eth2wrap.C
 	)
 	go func() {
 		ticker := clock.NewTicker(10 * time.Second)
+		peerCountTicker := clock.NewTicker(1 * time.Minute)
 		epochTicker := clock.NewTicker(32 * 12 * time.Second) // 32 slots * 12 second slot time
+		var (
+			bnPeerCount    int   // Beacon node peer count value which is queried on startup & every minute
```

Comment:
slightly simpler, lets use a `bnPeerCount *bool`,  then below use `if bnPeerCount != nil && *bnPeerCount == 0`

Reply:
i suppose you mean `bnPeerCount *int`